pyEpiabm/pyEpiabm/intervention/vaccination.py

```diff
@@ -0,0 +1,44 @@
+#
+# Vaccination Class
+#
+
+from pyEpiabm.intervention import AbstractIntervention
+
+
+class Vaccination(AbstractIntervention):
+    """Vaccination intervention
+    """
+
+    def __init__(
+        self,
+        daily_doses,
+        population,
+        **kwargs
+    ):
+        """ Set the parameters for vaccinations.
+
+        Parameters
+        ----------
+        daily_doses : int
+            Number of vaccine doses administered per day nationwide
+        """
+        self.daily_doses = daily_doses
+
+        # start_time, policy_duration, threshold, population
```

Comment:
If this comment refers to the kwargs, then make this clear (and remove the population)